its/src/test/java/com/sonar/it/csharp/NoSourcesTest.java

```diff
@@ -0,0 +1,50 @@
+package com.sonar.it.csharp;
```

Comment:
Java ITs failed on missing licence header in this file

Reply:
Indeed,Ionly ranit in theide